Home_Work_1/task7.py

```diff
@@ -0,0 +1,30 @@
+# Пользователь вводит число от 1 до 999. Используя операции с числами
+# сообщите что введено: цифра, двузначное число или трёхзначное число.
+# Для цифры верните её квадрат, например 5 - 25
+# Для двузначного числа произведение цифр, например 30 - 0
+# Для трёхзначного числа его зеркальное отображение, например 520 - 25
+# Если число не из диапазона, запросите новое число
+# Откажитесь от магических чисел
+# В коде должны быть один input и один print
+
+
+
+res = 0
+check = True
+
+while check:
+    number = int(input('Введите число от 1 до 999: '))
+    if number > 0 and number < 10:
```

Comment:
Как указано в условии задачи, нужно использовать операции с числами. Чтобы определить, что именно введено, а не логические выражения в виде математических неравенств.

Reply:
Используйте двойное неравенство.
```suggestion
    if 0 < number < 10:
```